srcpkgs/gcc/template

```diff
@@ -116,7 +117,7 @@ case "$XBPS_TARGET_MACHINE" in
 		subpackages+=" libquadmath libquadmath-devel"
 		subpackages+=" libvtv libvtv-devel"
 		;;
-	ppc64le)
+	ppc64|ppc64le)
```

Comment:
This is actually wrong. It was a try but with `libquadmath` enabled I cannot cross compile gcc from x86_64.
It is not that this is actually required AFAICT so please remove the `ppc64|` again.

Reply:
Alright. I'm recompiling gcc right now to investigate the cause of that, but I'll probably end up removing it then.